.github/templates/build-container/action.yaml

```diff
@@ -0,0 +1,36 @@
+name: build
+
+inputs:
+  dockerfile:
+    required: true
+  image:
+    required: true
+  registry:
+    required: true
+  user:
+    required: true
+  secret:
+    required: true
+
+runs:
+  using: "composite"
+  steps:
+    - name: Checkout the repository
+      uses: actions/checkout@master
+    - name: Calculate temporary variables
+      shell: sh
+      run: echo "SHORT_SHA=`git rev-parse --short HEAD`" >> $GITHUB_ENV
```

Comment:
What does it do?

Reply:
Calculates the short SHA of this commit. It's easier to read and use than the long one